### ISSUE TITLE
marketplace binding cannot be loaded 

### ISSUE THREAD

**Issue by mvalla** [openhab/openhab-core#2673]
In Eclipse IDE, if I try to start OH from latest 3.3.0-SNAPSHOT, checked out the app.bndrun from the main branch, I receive this error and the Marketplace binding is not started:

```
19:07:45.751 ERROR [main                ] [internal.json.JsonAddonService:206  ] - bundle org.openhab.core.addon.marketplace:3.3.0.202201070311 (111)[org.openhab.core.addon.marketplace.internal.json.JsonAddonService(42)] : Error during instantiation of the implementation object java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.felix.scr.impl.inject.internal.ComponentConstructorImpl.newInstance(ComponentConstructorImpl.java:316)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:286)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:115)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:1000)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:973)
19:07:50.534 ERROR [main                ] [internal.json.JsonAddonService:206  ] - bundle org.openhab.core.addon.marketplace:3.3.0.202201070311 (111)[org.openhab.core.addon.marketplace.internal.json.JsonAddonService(42)] : Error during instantiation of the implementation object java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.apache.felix.scr.impl.inject.internal.ComponentConstructorImpl.newInstance(ComponentConstructorImpl.java:316)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:286)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:115)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:1000)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:973)
19:07:50.744 WARN  [main                ] [.internal.addons.AddonResource:185  ] - bundle org.openhab.core.io.rest.core:3.3.0.202201070308 (134)[org.openhab.core.io.rest.core.internal.addons.AddonResource(145)] : Could not get service from ref {org.openhab.core.addon.AddonService}={service.config.factory=false, service.id=142, service.bundleid=111, service.scope=bundle, service.config.category=system, service.pid=org.openhab.jsonaddonservice, service.config.label=Json 3rd Party Add-on Service, component.name=org.openhab.core.addon.marketplace.internal.json.JsonAddonService, service.config.description.uri=system:jsonaddonservice, component.id=42}
19:07:54.951 INFO  [main                ] [e.internal.ModelRepositoryImpl:100  ] - Loading model 'demo.items'
```

Similar error is reported here in the community:
https://community.openhab.org/t/new-binding-marketplace-oh-3-2-floods-server-leads-to-a-http-429-response/130653

**Comment by mvalla** [comment]
Thank you for moving here, in fact I was not sure if it was an openhab-distro or openhab-core issue.

**Comment by wborn** [comment]
Fixed by #2678
